edx_rest_framework_extensions/middleware.py

```diff
@@ -164,3 +166,13 @@ def _cache_if_authenticated_user_found_in_middleware(self, request, value):
 
         if hasattr(request, 'user') and request.user and request.user.is_authenticated:
             DEFAULT_REQUEST_CACHE.set(self.AUTHENTICATED_USER_FOUND_CACHE_KEY, value)
+
+
+class RequestMetricsMiddleware(RequestCustomAttributesMiddleware):
+    """
+    Deprecated class for handling middleware. Class has been renamed to RequestCustomAttributesMiddleware.
+    """
+    def __init__(self, *args, **kwargs):
+        super(RequestMetricsMiddleware, self).__init__(*args, **kwargs)
+        msg = "Use 'RequestCustomAttributesMiddleware' in place of 'RequestMetricsMiddleware'."
```

Comment:
Can we print where this instantiation is being called from or will that come through some other way in the logs?

Reply:
Simply searching for the name made it very easy to find, so I don't think we need to add anything else.  See https://github.com/edx/edx-platform/pull/25082/files